src/nuclio/projects/project/functions/version/version-code/version-code.component.js

```diff
@@ -104,6 +104,26 @@
                         }
                     }
                 }
+            },
+            {
+                id: 's3',
+                visible: true,
+                name: 'S3',
+                defaultValues: {
+                    spec: {
+                        build: {
+                            code_entry_attributes: {
+                                s3_bucket: '',
+                                s3_item_key: '',
+                                s3_access_key_id: '',
+                                s3_secret_access_key: '',
+                                s3_region: '',
+                                s3_session_token: '',
+                                work_dir: ''
```

Comment:
I'm sorry I confused you.
In Nuclio open-source, these should be in camelCase:
```js
{
    codeEntryAttributes: {
        s3Bucket: '',
        s3ItemKey: '',
        s3AccessKeyId: '',
        s3SecretAccessKey: '',
        s3Region: '',
        s3SessionToken: '',
        workDir: ''
    }
}
```
Only in Iguazio Dashboard repo the `NuclioHelperService` should convert these to snake_case.